src/Classes/CategoryHydrator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Furniture\Classes;
```

Comment:
given that your going to have multiple hydrators, multiple entities and other classes, i would suggest splitting your classes into a `Furniture\Hydrators` namespace and `Furniture\Entities` etc etc